inc/db.function.php

```diff
@@ -1705,15 +1705,17 @@ function getEntitiesRestrictRequest($separator="AND", $table="", $field="",$valu
       $field = "`$table`.`$field`";
    }
 
-   $query .= "$field";
-
    if (is_array($value)) {
-      $query .= " IN ('" . implode("','",$value) . "') ";
+      $query .= "$field IN ('" . implode("','",$value) . "') ";
    } else {
       if (strlen($value) == 0) {
-         $query .= " IN (".$_SESSION['glpiactiveentities_string'].") ";
+         if (isset($_SESSION['glpiactiveentities_string'])) {
```

Comment:
I do not thing this is the way to work...

I guess that is entities string is not set; user is not logged correctly, and his entities are not loaded; I'd prefer the auth side to be fixed.

If entities are not loaded, there will be no restrictions at all, and user may show things from entities he's not part of.